tools/text_processing/split_file_to_collection/split_file_to_collection.xml

```diff
@@ -50,7 +50,7 @@
     </requirements>
     <command detect_errors="aggressive"><![CDATA[
         mkdir ./out &&
-        python '$__tool_directory__/split_file_to_collection.py'
+        '$__tool_directory__/split_file_to_collection.py'
```

Comment:
please leave python here ...

Reply:
okidoki. any specific reason for that? since the python script has a shebang line I thought this is OK.

Reply:
We agreed on this standard in IUC, I guess is easier to reason about it in this way.